src/OpenConext/AttributeAggregationApiClientBundle/Resources/config/services.yml

```diff
@@ -1,18 +1,9 @@
 services:
     openconext_aa_api.guzzle:
-        public: false
         class: GuzzleHttp\Client
         arguments:
             - [] # Client options
 
-    openconext_aa_api.service.api:
-        public: false
-        class: OpenConext\AttributeAggregationApiClientBundle\Http\JsonApiClient
+    OpenConext\AttributeAggregationApiClientBundle\Http\JsonApiClient:
         arguments:
-            - @openconext_aa_api.guzzle
-
-    openconext_aa_api.repository.attribute_aggregation:
-        public: false
-        class: OpenConext\AttributeAggregationApiClientBundle\Repository\AttributeAggregationRepository
-        arguments:
-            - @openconext_aa_api.service.api
\ No newline at end of file
+            - '@openconext_aa_api.guzzle'
```

Comment:
Missing a new line at the end of the file.

This happens in a few other places as well. How about adding an `.editorconfig` in another PR to reduce these kinds of errors?

For more info see [their website](https://editorconfig.org/)

Reply:
Added the newline, but not yet added an editorconfig.